src/sql-parser/src/parser.rs

```diff
@@ -2337,6 +2337,11 @@ impl<'a> Parser<'a> {
                     compression,
                 })
             }
+            LOAD => {
+                self.expect_keyword(GENERATOR)?;
+                let generator = self.parse_literal_string()?;
```

Comment:
I'd say we should make this a keyword rather than a string literal. I imagine we'll want to add all kinds of options to the more complicated load generators, and we've been moving in the direction of making those options actual SQL keywords with parser support rather than unstructured `WITH` options.

Reply:
Actually, it seems like even this load generator could benefit from a `TICK INTERVAL` option!